src/syntax-tree/ast.ts

```diff
@@ -2013,8 +2019,10 @@ export class FunctionCallStmt extends Statement implements Importable {
     }
 
     validateImportFromImportList(imports: ImportStatement[]): boolean {
-        const relevantImports = imports.filter((stmt) => stmt.getImportModuleName() === this.requiredModule);
-        console.log(relevantImports);
+        const relevantImports = imports.filter(
```

Comment:
We have a choice of either filtering based on line number before we call this method, but those places don't necessarily have line-number information or don't necessarily need to check it. This is way safer so that even if we forget to validate on line numbers when populating `imports`, this method will still run the check making it safe.